.packit.yml

```diff
@@ -76,34 +100,9 @@ jobs:
     dependencies:
       - anaconda-webui
     dist_git_branches:
-      - main
+      - fedora-development
```

Comment:
Aren't we losing Rawhide builds here?

Reply:
https://packit.dev/docs/configuration#aliases
fedora-rawhide is always under developent.